.asf.yaml

```diff
@@ -0,0 +1,39 @@
+  
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: An observability database aims to ingest, analyze and store Metrics, Tracing and Logging data.
+  homepage: https://skywalking.apache.org/
+  labels:
+    - skywalking
+    - observability
+    - apm
+    - database
+  enabled_merge_buttons:
+    squash:  true
+    merge:   false
+    rebase:  false
+  protected_branches:
+    main:
+      required_status_checks:
+        strict: true
+        contexts:
+          - Continuous Integration
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        required_approving_review_count: 1
```

Comment:
> @hanahmily I still keep the main branch unprotected, direct push is still working.

This is not the case you said

Reply:
Is this going to add branch protection? This was for review requirement

Reply:
> Is this going to add branch protection? This was for review requirement

This config `strict: true` protects the branch.

Reply:
> > Is this going to add branch protection? This was for review requirement
> 
> This config `strict: true` protects the branch.

And this

```
contexts:
  - Continuous Integration
```

According to GitHub

> - [x] Require status checks to pass before merging
> Choose which status checks must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed.